cls/SourceControl/Git/WebUIDriver.cls

```diff
@@ -340,4 +342,10 @@ ClassMethod GetGitVersion() As %Library.DynamicObject
     quit {"version": (version)}
 }
 
+ClassMethod GetHomeURL() As %SystemBase
+{
+    set homeURL = "/csp/sys/UtilHome.csp"
```

Comment:
We might need to put a bit more intelligence in here for instances with a CSPInstanceName setup. Let's add a separate method that returns this URL which is called both here...